src/python/install.sh

```diff
@@ -454,6 +454,20 @@ if [[ "${INSTALL_PYTHON_TOOLS}" = "true" ]] && [[ $(python --version) != "" ]];
             echo "${util} already installed. Skipping."
         fi
     done
+    
+    # Temporary: Removes “setup tools” metadata directory due to https://github.com/advisories/GHSA-r9hx-vwmv-q579
+
+    VULNERABLE_VERSIONS=("3.10" "3.11")
+    RUN_TIME_PY_VER_DETECT=$(python --version 2>&1)
+    PY_MAJOR_MINOR_VER=${RUN_TIME_PY_VER_DETECT:7:4};
+
+    for vv in "${VULNERABLE_VERSIONS[@]}"; do
+        if [ "${PY_MAJOR_MINOR_VER}" == "${vv}" ]; then
+            rm -rf  ${PIPX_HOME}/shared/lib/"python${vv}"/site-packages/setuptools-65.5.0.dist-info
+            rm -rf ${PYTHON_INSTALL_PATH}/"${VERSION}"/lib/python${vv}/site-packages/setuptools-65.5.0.dist-info
```

Comment:
Can you help answer this comment in https://github.com/devcontainers/features/pull/809#discussion_r1468185812? 

> Do you have a scenario or reproduction for this?

As far as I remember, for the `mcr.microsoft.com/devcontainers/python` image, we were seeing `setuptools` only within `./usr/local/lib/python3.10/ensurepip/_bundled/setuptools-65.5.0-py3-none-any.whl` and `./usr/local/py-utils/shared/lib/python3.10/site-packages/setuptools-65.5.0.dist-info`. Hence, I'd like to understand why we want to remove from L:467 as well.

Reply:
Great, thanks!